libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -649,6 +657,15 @@ void AP_MotorsMulticopter::output_logic()
             break;
         }
 
+        // check whether it's safe to go to throttle unlimited
+        if (!check_failsafe()) {
```

Comment:
this will kill quadplanes on back transition

Reply:
also could break throw mode?

Reply:
this low level AP_Motors code should not take any actions when it discovers the problem. It should detect it only, then high level vehicle code needs to be the one to take the action

Reply:
Ok, fixed